spec/rails_helper.rb

```diff
@@ -14,14 +16,12 @@
 end
```

Comment:
Remove everything for stripe and format code

Reply:
Done!